<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlatformList</class>
 <widget class="QWidget" name="PlatformList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Platform List</string>
  </property>
  <widget class="QListWidget" name="platListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>34</y>
     <width>243</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="add_plat_but">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>413</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add platform</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_plat_but">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>413</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Remove platform</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plist_save_but">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>442</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_plat_but">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>442</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edit platform</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="plat_enabled_check">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>8</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Platform enabled?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
